core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss

```diff
@@ -1 +1,50 @@
-//TODO: implement image styles
+@use "variables" as *;
+
+.p-component-overlay.p-image-mask {
+    background-color: $black;
+}
+
+.p-image-preview-indicator {
+    background-color: transparent;
+    color: $white;
+
+    .pi-window-maximize {
+        font-size: 2rem;
+    }
+}
+
+.p-image-preview-container:hover > .p-image-preview-indicator {
+    background-color: rgba(0, 0, 0, 0.5);
+}
+
+.p-image-toolbar {
+    padding: $spacing-3;
+}
+
+.p-image-action.p-link {
+    color: $white;
+    background-color: transparent;
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
```

Comment:
Does exist a css-var for the border radius?

Reply:
no but just create `$border-radius-circular`